### ISSUE TITLE
Masked Language Modeling and the Distributional Hypothesis: Order Word Matters Pre-training for Little

### ISSUE THREAD

**Issue by ymgw55** [ymgw55/MLPapers#126]
### URL
https://arxiv.org/abs/2104.06644#
### Authors
Koustuv Sinha, Robin Jia, Dieuwke Hupkes, Joelle Pineau, Adina Williams, Douwe Kiela
### Published at
Facebook AI Research
### Date of issue
9 Sep 2021

# Reading Motivation
紹介してもらった論文
# Summary
mlmのパフォーマンスの説明として, 「モデルが classical な nlp での文構造を学習している」といったものがある. 著者らは, 異なる説明として, mlmがdown strem タスクでworkするのは, 高次元での単語の共起情報をモデリングする能力によると考える. これを実証するために, pre-train時に語順をシャッフルした文を用い, そのような場合でもfine-tuning後にモデルが高い精度を達成することを示す. 精度検証のdown stream task にはPAWSなどの語順sensitiveなtaskも含む. 著者らのモデルはいくつかの文構造に関するprobing test でも, 驚くほど良い性能を発揮したので, 構造情報表現のテスト方法に問題がある可能性を示唆している. 結論として, 単純な単語の分布情報で pre-training の成功の大部分を説明でき, 同時により深い言語知識を必要とする, challengingなデータセットを精査することの重要性を強調している.

# Experiment
M_N: RoBERTa
M_n: 入力文について, n-gram単位で元の位置にならないようにシャッフル
M_UG: コーパスから, 頻度分布で uni-gram をサンプリング
M_NP: no positional encoding
M_RI: モデルパラメータの random initialized

fine-tuning は全てnatural word order.
![Screenshot from 2021-11-30 11-04-19](https://user-images.githubusercontent.com/39965001/143972629-8b57b629-5e3b-4323-b4ec-c59476cf3a50.png)

各モデルに対し, fine-tuning 時に natural word order, random word order で比較.
![Screenshot from 2021-11-30 11-04-36](https://user-images.githubusercontent.com/39965001/143972639-6e7e9c97-19f8-470f-b60f-ca4828e6a566.png)

# Memo
PAWS は人口データなのでそもそも語順を測るタスクになっているのか? #131
